test/l2SharedBridge/L2SharedBridgeBase.t.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import {Test} from "forge-std/Test.sol";
+import {MockERC20} from "forge-std/mocks/MockERC20.sol";
+import {L2SharedBridge} from "../../src/L2SharedBridge.sol";
+import {IL1ERC20Bridge} from "../../src/interfaces/IL1ERC20Bridge.sol";
+import {IL2Messenger} from "../../src/interfaces/IL2Messenger.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {TransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import {IL2SharedBridge} from "../../src/interfaces/IL2SharedBridge.sol";
+
+contract MintableToken is MockERC20 {
```

Comment:
I don't know how similar this is to the real USDC, it might be good enough

Reply:
> but it might make sense to test it with the full thing
what do you mean by the full thing? i think we only need mint and burn to test